python/setup.py

```diff
@@ -22,38 +22,37 @@
 
 """setup.py"""
 import os
-from setuptools import setup
-
-try: # for pip >= 10
-    from pip._internal.req import parse_requirements
-    from pip._internal.download import PipSession
-except ImportError: # for pip <= 9.0.3
-    from pip.req import parse_requirements
-    from pip.download import PipSession
-from optparse import Option
+import pathlib
+import pkg_resources
+import setuptools
+from setuptools import find_packages, setup
 
 BASEDIR = os.path.dirname(os.path.abspath(__file__))
 
 
 def parse_reqs(reqs_file):
-    ''' parse the requirements '''
-    options = Option("--workaround")
-    options.skip_requirements_regex = None
-    options.isolated_mode = True
-    install_reqs = parse_requirements(reqs_file, options=options, session=PipSession())
-    return [str(ir.req) for ir in install_reqs]
+    """ parse the requirements """
+    install_reqs = list()
+    with pathlib.Path(reqs_file).open() as requirements_txt:
+        install_reqs = [
+            str(requirement)
+            for requirement in pkg_resources.parse_requirements(requirements_txt)
+        ]
+
+    return install_reqs
 
 
 REQS = parse_reqs(os.path.join(BASEDIR, "requirements.txt"))
 
+
 # read the contents of README.md
 this_directory = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(this_directory, 'README.md')) as f:
+with open(os.path.join(this_directory, "README.md")) as f:
     long_description = f.read()
```

Comment:
I don't think you need to append "this_directory"

```python
with open('README.md') as f:
    long_description = f.read()
```
Seems to be enough in MorphIO.